quantestpy/test_circuit.py

```diff
@@ -683,152 +361,99 @@ def _get_whole_gates(self,) -> np.ndarray:
         for gate in self._gates:
             if gate["name"] == "id":
                 all_qubit_gate = \
-                    self._create_all_qubit_gate_from_single_qubit_gate(
-                        _ID, gate["target_qubit"])
+                    self._create_all_qubit_gate_from_original_qubit_gate(
+                        _ID, gate["control_qubit"],
+                        gate["target_qubit"], gate["control_value"])
 
-            elif gate["name"] == "x":
+            elif gate["name"] in ("x", "cx"):
                 all_qubit_gate = \
-                    self._create_all_qubit_gate_from_single_qubit_gate(
-                        _X, gate["target_qubit"])
+                    self._create_all_qubit_gate_from_original_qubit_gate(
+                        _X, gate["control_qubit"],
+                        gate["target_qubit"], gate["control_value"])
 
```

Comment:
@mitz1012 
Sorry that I did not realize this earlier, but calling `self._create_all_qubit_gate_from_original_qubit_gate()` in every if-sentence is a waste of space and even might induce a mistake. The following would be easier to read, wouldn't it?

```py
if gate["name"] == "id":
    original_qubit_gate = _ID

elif gate["name"] in ("x", "cx"):
    original_qubit_gate = _X
    
    ...

elif gate["name"] == "scalar":
    original_qubit_gate = _scalar(gate["parameter"])

else:
    raise

all_qubit_gate = self._create_all_qubit_gate_from_original_qubit_gate(
                        original_qubit_gate, 
                        gate["control_qubit"],
                        gate["target_qubit"], 
                        gate["control_value"]
)
```

Reply:
Thanks! I have totally agreed your suggestion. Check `test_circuit.py` again please!